### PR TITLE
docs(0010): describe current path choosing algorithm

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 1]
```diff
@@ -28,18 +28,7 @@ Note that the points series ("liquidity curves") in route broadcasts, as well as
...
-in the bill in a restaurant, on top of the price that was quoted in the menu.
-
-Although this strategy works well in closed networks where each participant is vetted and can be trusted, it is known to be sub-optimal for use in an open network where connectors
-may try to earn extra money by charging a higher fee than the cost price. Therefore, this "gratuity quoting" strategy used by the ilp-kit/ilp-connector reference implementation is
-likely to change in the future, and one proposal is currently being developed.
-
-### Next version: Liquidity Routing
-In Liquidity Routing, the curve from the route broadcast is used directly for determining a source amount / destination amount relation, and remote quoting is only used when
+The curve from the route broadcast is used directly for determining a source amount / destination amount relation, and remote quoting is only used when
 a (default) route was configured without curve information. This requires routes to be re-broadcast when their liquidity curve changes, but ilp-kit will aggregate over time, both
```
Comment (sentientwaffle) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: > remote quoting is only used when a (default) route was configured without curve information

It also uses remote quoting if a route is _received_ without a curve.

Comment (michielbdejong) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: right, true!

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 2]
```diff
@@ -28,19 +28,12 @@ Note that the points series ("liquidity curves") in route broadcasts, as well as
...
-
-### Next version: Liquidity Routing
-In Liquidity Routing, the curve from the route broadcast is used directly for determining a source amount / destination amount relation, and remote quoting is only used when
-a (default) route was configured without curve information. This requires routes to be re-broadcast when their liquidity curve changes, but ilp-kit will aggregate over time, both
+The curve from the route broadcast is used directly for determining a source amount / destination amount relation, and remote quoting is only used when
+a connector knows about a route without knowing its curve information (this can happen if a connector somewhere in the network was explicitly configured with a hard-wired
+(default) route in its ilp-connector config; that route then also gets forwarded without curve in route broadcasts across the network, and all connectors that want to use this
+route will therefore need to use remote quoting).
+All routes that were built up from local pairs will however get broadcast including their curves, based on connector fees, currency exchange rates, and liquidity limits (min and
```
Comment (sentientwaffle) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: > All routes that were built up from local pairs will however get broadcast including their curves

Connectors have the option of broadcasting routes without curves (`CONNECTOR_BROADCAST_CURVES`).

Comment (michielbdejong) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: yes, but that's a non-default config option. They also have the option to peer over five-bells, among other things :)

**Pull request merged**